src/server_manager/web_app/ui_components/app-root.js

```diff
@@ -367,15 +404,30 @@ export class AppRoot extends mixinBehaviors
           </div>
 
           <!-- Links section -->
-          <paper-listbox on-tap="maybeCloseDrawer">
-            <a href="https://s3.amazonaws.com/outline-vpn/index.html#/en/support/dataCollection">[[localize('nav-data-collection')]]</a>
+          <paper-listbox>
+            <span on-tap="maybeCloseDrawer"><a href="https://s3.amazonaws.com/outline-vpn/index.html#/en/support/dataCollection">[[localize('nav-data-collection')]]</a></span>
             <span on-tap="submitFeedbackTapped">[[localize('nav-feedback')]]</span>
-            <a href="https://s3.amazonaws.com/outline-vpn/index.html#/en/support/">[[localize('nav-help')]]</a>
+            <span on-tap="maybeCloseDrawer"><a href="https://s3.amazonaws.com/outline-vpn/index.html#/en/support/">[[localize('nav-help')]]</a></span>
             <span on-tap="aboutTapped">[[localize('nav-about')]]</span>
-            <div class="legal-links">
-              <a href="https://www.google.com/policies/privacy/">[[localize('nav-privacy')]]</a>
-              <a href="https://s3.amazonaws.com/outline-vpn/static_downloads/Outline-Terms-of-Service.html">[[localize('nav-terms')]]</a>
-              <span on-tap="showLicensesTapped">[[localize('nav-licenses')]]</span>
+            <div id="links-footer">
+              <paper-icon-item id="language-row">
```

Comment:
It seems like you could factor out this component, in order to not grow app-root further. Maybe use LitElement?

Reply:
Converted it to Polymer temporarily. Will look into converting to LitElement.